Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/install/windows.md

<admon type="tip">

To use DVC [as a Python library](/doc/api-reference), you can install it
[with `conda`](#install-with-conda) or [with `pip`](#install-with-pip).

</admon>

<admon type="tip">

> You can find important tips to improve your experience using DVC on Windows 
> by reviewing [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows).  

Reviewer: ```suggestion
See [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows)
for important tips to improve your experience on Windows.
```
